### PR TITLE
Fix Mobile Vikings MNC code was not correct so link fot more info on …

### PR DESCRIPTION
**Pull request by ruditimmermans**
…the Mobile Vikings website.

https://support.mobilevikings.be/hc/nl/articles/202836041-Ik-heb-geen-mobiel-internet-Wat-moet-ik-doen

### CODE REVIEW

**Review by pvuorela — APPROVED** [review]
Based on forum and the mobile vikings website there seems to be a bit conflicting information, but none having the current mnc 20. If this gets it working for you, I'm assumign it's for the better.

File: mobile-broadband-provider-info/serviceproviders.xml [hunk 1]
```diff
@@ -1266,13 +1266,14 @@ conceived.
...
-			<network-id mcc="206" mnc="20"/>
+			<network-id mcc="206" mnc="01"/>
 			<voicemail>1933</voicemail>
-			<apn value="web.be">
+			<apn value="internet.proximus.be">
 				<plan type="prepaid"/>
 				<usage type="internet"/>
 				<username>web</username>
 				<password>web</password>
+				<name>Mobile Vikings Internet</name>
```
Comment (pvuorela) [inline mobile-broadband-provider-info/serviceproviders.xml]: This needs to be below the `<usage>`. Otherwise the xml doesn't validate and build fails.

**Review by pvuorela — CHANGES_REQUESTED** [review]

**Review by mlehtima — APPROVED** [review]

**Review by pvuorela — COMMENTED** [review]
Looks better now, but could you still rebase and squash the commits to avoid intermediate steps in the git history. I.e. git rebase -i origin/master and mark the latter commit with 's'.

**Comment by ruditimmermans** [comment]
> Looks better now, but could you still rebase and squash the commits to avoid intermediate steps in the git history. I.e. git rebase -i origin/master and mark the latter commit with 's'.

I'm sorry i work directly online via the Github website and i never used Git itself on Windows so this is my first time i have dont something like this sorry....

**Comment by pvuorela** [comment]
> I'm sorry i work directly online via the Github website and i never used Git itself on Windows so this is my first time i have dont something like this sorry....

Ah, interesting. But no worries, I can also do some squashing with the merging. Just commonly better if PR author handles that.

**Review by pvuorela — APPROVED** [review]

**Comment by ruditimmermans** [comment]
> > I'm sorry i work directly online via the Github website and i never used Git itself on Windows so this is my first time i have dont something like this sorry....
> 
> Ah, interesting. But no worries, I can also do some squashing with the merging. Just commonly better if PR author handles that.

Ok i understand i will look on git how this works for into the future i do something myself, thanks for the helps!

**Comment by ruditimmermans** [comment]
Hi Pekka,

I was into a talk with Jussi into a private message on the forum and we found out my mobile network would be crashing with Offono i send the offono logs back to Jussi but i dont hear anything more, would i need to send the logs to you also or someone elsse ?

Thanks!
Rudi

**Pull request merged**